webapp/channels/src/components/user_settings/security/user_settings_security.test.tsx

```diff
@@ -110,17 +114,17 @@ describe('components/user_settings/display/UserSettingsDisplay', () => {
             enableOAuthServiceProvider: true,
         };
 
-        const wrapper = shallow<UserSettingsSecurity>(<UserSettingsSecurity {...props}/>);
+        const wrapper = shallow<SecurityTab>(<SecurityTab {...props}/>);
 
         await promise;
 
         expect(wrapper.state('serverError')).toEqual(error.message);
     });
 
     test('submitPassword() should not have called updateUserPassword', async () => {
-        const wrapper = shallow(<UserSettingsSecurity {...requiredProps}/>);
+        const wrapper = shallow(<SecurityTab {...requiredProps}/>);
 
-        await (wrapper.instance() as UserSettingsSecurity).submitPassword();
+        await (wrapper.instance() as SecurityTab).submitPassword();
```

Comment:
Out of the scope:
To avoid the `as`, you can do `const wrapper = shallow<SecurityTab>(<SecurityTab {...requiredProps}/>);`

Similar in other places in this file.